src/plugins.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+
+export const getPluginTab = tab => (
+    React.lazy(async () => {
+        try {
+            return import(`plugins/${tab}/component.jsx`);
+        } catch (error) {
+            return import(`./tabs/${tab}`);
+        }
+    })
+);
```

Comment:
This does the magic of loading the component of fallback. In this case there is no need for fallbacks, but it's possible in a case where you'd want an extension point, but still have some default behaviour.